example/src/getTests.ts

```diff
@@ -660,6 +660,20 @@ export function getTests(
     createTest('promiseThrows() throws', async () =>
       (await it(() => testObject.promiseThrows())).didThrow()
     ),
+    createTest('twoPromises can run in parallel', async () =>
+      (
+        await it(async () => {
+          const start = performance.now()
+          // 2s + 2s = ~4s in serial, ~2s in parallel
+          await Promise.all([testObject.wait(2), testObject.wait(2)])
```

Comment:
I think a better test case would be to check how long did each promise take to resolve.

So, make wait 4, wait 2, and make sure first one took 4s, second one 2s to resolve.

What you'll find out, unless your rewrite did fix things, is that they will both take same time to resolve (4s)